.github/workflows/ci.yml

```diff
@@ -0,0 +1,334 @@
+name: 'CI'
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *"
+
+env:
+  APPLICATION_ENV: ci.pgsql
+  SPRYKER_TESTING_ENABLED: 1
+  APPLICATION_STORE: DE
+  PROJECT: suite
+  POSTGRES_PORT: 5433
+  POSTGRES_PASSWORD: postgres
+  POSTGRES_USER: postgres
+  POSTGRES_DB: DE_test_zed
+  MYSQL_PASSWORD: secret
+  MYSQL_USER: root
+  MYSQL_DATABASE: DE_test_zed
+
+jobs:
+  validation-7-4:
+    name: 'Static analysis / PHP 7.4'
+    runs-on: ubuntu-18.04
+    services:
+      postgres:
+        image: postgres:12
+        env:
+          POSTGRES_PASSWORD: ${{ env.POSTGRES_PASSWORD }}
+          POSTGRES_USER: ${{ env.POSTGRES_USER }}
+          POSTGRES_DB: ${{ env.POSTGRES_DB }}
+        ports:
+          - 5433:5432
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+    steps:
+      - uses: actions/checkout@v2
+      - uses: shivammathur/setup-php@v2
+        with:
+          php-version: '7.4'
+          ini-values: memory_limit=-1
+      - name: 'Configure sysctl limits'
+        run: |
+          sudo swapoff -a
+          sudo sysctl -w vm.swappiness=1
+          sudo sysctl -w fs.file-max=262144
+          sudo sysctl -w vm.max_map_count=262144
+
+      - name: 'Runs Elasticsearch'
+        uses: elastic/elastic-github-actions/elasticsearch@master
+        with:
+          stack-version: 6.8.0
+      - name: 'Prepare code base'
+        run: ./config/Shared/ci/ga/prepare_for_validation.sh
+      - name: 'Test'
+        run: ./config/Shared/ci/ga/validate.sh
+
+  acceptance-7-4:
+    name: 'Acceptance tests / PHP 7.4'
+    runs-on: ubuntu-18.04
+    services:
+      rabbitmq:
+        image: rabbitmq
+        ports:
+          - 5672:5672
```

Comment:
if ports are the same, I guess you don't need to map them

Reply:
fixed

Reply:
But it breaks workflow. According https://docs.github.com/en/actions/guides/about-service-containers#mapping-docker-host-and-service-container-ports